docs/source/guides/libraries.rst

```diff
@@ -301,12 +272,313 @@ Python Library
 Where to Get
 ^^^^^^^^^^^^
 
+Prerequisites
+"""""""""""""
+
+- CMake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher)
+- swig(3.0.12 can be built --without-pcre)
+- protobuf
+- python (python-dev)
+
+Install Iroha python libraries:
```

Comment:
Please use "Title Case" style for headings. Details and the tool for checks - https://capitalizemytitle.com/ (no additional settings required)

Reply:
Please remove colon from headings